packages/ui/src/theme/tokens/components/index.ts

```diff
@@ -60,7 +105,6 @@ export const components = {
   loader,
   menu,
   pagination,
-  phonenumberfield,
```

Comment:
Why are we removing `phonenumberfield`?

Reply:
Because it doesn't have a single token in it. :)